modules/test/playwright/tests/frontend-editor-ckeditor-web/main/tests/ckeditor4/classic.spec.ts

```diff
@@ -161,15 +164,13 @@ test(
 		});
 
 		await expect(ckeditor4Page.contextMenu.getByLabel('URL')).toHaveValue(
-			'/documents/d/guest/satellite-png'
+			/satellite-png/
```

Comment:
Fix of an unrelated test failure, there are several like this in PR. Failures were caused by change of internal URLs of document library,